pkgs/by-name/vi/vim-language-server/package.nix

```diff
@@ -25,20 +26,12 @@ stdenv.mkDerivation (finalAttrs: {
 
   nativeBuildInputs = [
     yarnConfigHook
+    yarnBuildHook
     # Needed for executing package.json scripts
```

Comment:
Maybe the hook should check if we have a node in PATH and if not exit with an explanation?

Reply:
> Maybe the hook should check if we have a node in PATH and if not exit with an explanation?

I'm not sure about this.. Are you positive that a `node` in `$PATH` will always be required for `yarn build`?

Reply:
Nope 😅 lets see how other packages are done and if it becomes a general pattern, we can still add it.

Reply:
> Nope 😅 lets see how other packages are done and if it becomes a general pattern, we can still add it.

BTW `patchShebangs` won't patch the shebangs without the a `node` in `$PATH`. I do plan to iterate all the rest of the 36 packages that use `mkYarnPackage`, so I'll keep this thread open and keep in mind to rethink about it after I finish iterating.